handlers/datasetviewing/show.go

```diff
@@ -90,10 +84,6 @@ func ShowPublications(w http.ResponseWriter, r *http.Request) {
 	datasetviews.ShowPublications(c, dataset, relatedPublications).Render(r.Context(), w)
 }
 
-func (h *Handler) ShowActivity(w http.ResponseWriter, r *http.Request, legacyCtx Context) {
-	render.View(w, "dataset/show_activity", YieldShowActivity{
-		Context:      legacyCtx,
-		SubNavs:      subNavs,
-		ActiveSubNav: "activity",
-	})
+func ShowActivity(w http.ResponseWriter, r *http.Request) {
+	datasetviews.ShowActivity(ctx.Get(r), ctx.GetDataset(r)).Render(r.Context(), w)
```

Comment:
Keep the name `ShowActivity` or set it to a similar name (just `Activity`) like in the publicationviews?

Reply:
just Activity

Reply:
done